src/api/Gemfile.lock

```diff
@@ -437,8 +437,7 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    selenium-webdriver (4.5.0)
-      childprocess (>= 0.5, < 5.0)
+    selenium-webdriver (4.16.0)
```

Comment:
Current version of selenium-webdriver, which is 4.5.0, doesn't support chilprocess version 5.0. Closing.